tests/src/Search/Web/ExampleWebTest.php

```diff
@@ -33,7 +33,7 @@ public function testElasticSearchIndex()
                 ],
```

Comment:
Yeah, was going to talk to you about testing. Wondering if `testSerializationSmokeTest()` should be expanded to test that it produces the same results as the SDK serialization? (Which raises the bigger problem that the SDK serialization should be used instead.)